### DISCUSSION TITLE
manifesto/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#38] [Announcements]
# manifesto/

This Is It.

https://www.supergoodcode.com/manifesto/

**Comment by NHOrus** [comment]
Mike Blumenkrantz is not real, he can't hurt us.

**Comment by datenwolf** [comment]
GLX deletion only if WGL is being deleted at the same time and both excuses of haphazard "quasi-WSI" APIs replaced with clean counterparts. On both ends this must include support for the long standing graphics APIs (X11 will never die. Win32 GDI will never die), so that legacy GLX/WGL wrappers can be built on the new-and-improved API


.


.


.


.


.


just to metastasize back into the original form ;-)

**Comment by BoyBaykiller** [comment]
Are you the chosen one that will finally bring Bindless Textures to core🙏
Also the OpenGL AMD team said they have plans to [support mesh shaders](https://community.amd.com/t5/opengl-vulkan/is-mesh-shader-support-planned-for-opengl/m-p/649970/highlight/true#M5068). I expect a cross vendor extension next month :)

Nah but good post and congratulations (i am only half joking).

**Comment by jams3223** [comment]
OMG it's Mike can I thought you abandoned us.

**Comment by qwertychouskie** [comment]
Look ma, I'm in a SGC post!

**Comment by K0bin** [comment]
> Native GL drivers on mobile platforms shall be outlawed!

This one is actually happening. Google is planning to use ANGLE on all Android devices at some point in the future.
https://developer.android.com/about/versions/15/features#gpu-api-roadmap

Pretty sure Apple is already using their OpenGL on Metal implementation.
